docker/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM ubuntu:24.04 AS aos-core-build-base
```

Comment:
Probably better to put in aos_core_common_cpp repo. Let's discuss.

Reply:
done